packages/react-components/react-utilities/src/utils/isHTMLElement.ts

```diff
@@ -0,0 +1,11 @@
+/**
+ * verifies if a given node is an HTMLElement,
+ * this method works seamlessly with frames and elements from different documents
+ */
+export function isHTMLElement(element: Node | null): element is HTMLElement {
```

Comment:
```suggestion
export function isHTMLElement(element: Node | null | undefined): element is HTMLElement {
```

nit: let's make this as easy to use as possible, sometimes it just so happens you might end up with a variable that could be `undefined`